projecteuler/problems/finds.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+from projecteuler.helpers import primenumbers, factors, number_checks
+
+def find_largest_prime_factor(number):
+    primes = primenumbers.get_prime_numbers(0, number)
+    if not primes:
+        return None
+    
+    all_factors = factors.get_factors(number)
+    if not all_factors:
+        return None
+
+    prime_factors = set(primes).intersection(all_factors)
+    return None if not prime_factors else max(prime_factors)
+
+def find_largest_palindromic_sum(min_number_to_sum, max_number_to_sum):
+    for max_number in range(max_number_to_sum, 0, -1):
```

Comment:
top marks for seeing "find largest" and iterating backwards. you'd be surprised how many people still iterate up from 0 ...